cdk/script/ci

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+source ${DIR}/setup
```

Comment:
I'd inline these into the `ci` file for simplicity. I think we'd also want to add `--frozen-lockfile` to `yarn install` in CI for a deterministic build.

From https://classic.yarnpkg.com/en/docs/cli/install/:

> If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass --frozen-lockfile flag.